src/main/steps/c100-rebuild/typeoforder/select-courtorder/content.ts

```diff
@@ -0,0 +1,218 @@
+    },
+    stopOtherPeopleDoingSomethingSubField: {
+      required: 'Specify what you want the court to the other people in the application doing',
+    },
+    resolveSpecificIssueSubField: {
+      required: 'Specify what issue you want the court to resolve',
+    },
+  },
+});
+
+const cy = () => ({
+  title: 'What are you asking the court to do? - welsh',
+  select_all_apply: 'Select all that apply - welsh',
+  whoChildLiveWith: 'Decide who the children live with and when - welsh',
+  childTimeSpent: 'Decide how much time the children spend with each person - welsh',
+  stopOtherPeopleDoingSomething: 'Stop the other people in the application doing something - welsh',
+  stopOtherPeopleDoingSomethingHint: 'For example, moving abroad or abducting the children - welsh',
+  resolveSpecificIssue: 'Resolve a specific issue you are concerned about - welsh',
+  resolveSpecificIssueHint: 'For example, what school the children will go to - welsh',
+  changeChildrenNameSurname: "Changing the children's names or surname - welsh",
+  changeChildrenNameSurname1: "Changing the children's names or surname - welsh",
+  allowMedicalTreatment: 'Allowing medical treatment to be carried out on the children - welsh',
+  takingChildOnHoliday: 'Taking the children on holiday - welsh',
+  relocateChildrenDifferentUkArea: 'Relocating the children to a different area in England and Wales - welsh',
+  relocateChildrenOutsideUk:
+    'Relocating the children outside of England and Wales (including Scotland and Northern Ireland) - welsh',
+  relocateChildrenDifferentUkArea1: 'Relocating the children to a different area in England and Wales - welsh',
+  relocateChildrenOutsideUk1:
+    'Relocating the children outside of England and Wales (including Scotland and Northern Ireland) - welsh',
+  specificHoliday: 'A specific holiday or arrangement - welsh',
+  whatSchoolChildrenWillGoTo: 'What school the children will go to - welsh',
+  religiousIssue: 'A religious issue - welsh',
+  medicalTreatment: 'Medical treatment - welsh',
+  returningChildrenToYourCare: 'Returning the children to your care - welsh',
+  returningChildrenToYourCareHint:
+    'If the children have been abducted, unlawfully removed or unlawfully retained - welsh',
+  errors: {
+    courtOrder: {
+      required: 'Select  what you are asking the court to do - welsh',
+    },
+    stopOtherPeopleDoingSomethingSubField: {
+      required: 'Specify what you want the court to the other people in the application doing - welsh',
+    },
+    resolveSpecificIssueSubField: {
+      required: 'Specify what issue you want the court to resolve - welsh',
+    },
+  },
+});
+
+const languages = {
+  en,
+  cy,
+};
+
+export const form: FormContent = {
+  fields: {
+    courtOrder: {
+      id: 'courtOrder',
+      type: 'checkboxes',
+      hint: l => l.select_all_apply,
+      validator: atLeastOneFieldIsChecked,
+      values: [
+        {
+          name: 'courtOrder',
```

Comment:
Hope we are saving the selected values into session, as we need to render next screens based on this selection.

Reply:
Yes, Framework does save in userCase automatically same as other screens